book/types_of_data.md

```diff
@@ -34,89 +32,31 @@ You can cast a string into an Decimal with the `into decimal` command
 
 ## Strings
 
-A string of characters that represents text. There are a few ways we can represent text in Nushell:
-
-**Double quotes**
-
-```
-"my message"
-```
-
-Double quotes are the most common form of quotes and one you may see whenever text is required.
-
-**Single quotes**
-
-```
-'my message'
-```
-
-Single quotes also give you a string value, just like double quotes. The difference here is that they allow you to use double quotes in the text: `'he said "can you grab my glass?"'`
-
-**String interpolation**
-Nushell supports string interpolation, allowing you to run sub-expressions inside of strings prefixed with `$`. For instance:
-
-```
-> echo $"6 x 7 = (6 * 7)"
-6 x 7 = 42
-```
-
-```
-> ls | each { |it| echo $"($it.name) is ($it.size)" }
-───┬─────────────────────
- 0 │ genawait is 4.1 KB
- 1 │ learncpp is 4.1 KB
- 2 │ nuscripts is 4.1 KB
-───┴─────────────────────
-```
-
-**Bare strings**
-
-```
-> echo hello
-```
-
-A unique trait of Nushell is that you can also create a string of one word without any quotes at all.
-
-The above is the same as if we had written:
-
-```
-> echo "hello"
-```
-
-Cast from a string into another type by using the `into <type>` command
-
-```
-> "1" | into int
-> "1.2" | into decimal
-```
-
-Also see [Handling Strings](https://www.nushell.sh/book/loading_data.html#handling-strings).
-
-## Lines
-
-Lines are strings with an implied OS-dependent line ending. When used, the OS-specific line ending is used.
-
-## Column paths
-
-Column paths are a path through the table to a specific sub-table, column, row, or cell.
+A string of characters that represents text. There are a few ways these can be constructed:
 
-Eg) The value `foo.0.bar` in `open data.toml | get foo.0.bar`
+- Double quotes
+  - `"Line1\nLine2\n"`
+- Single quotes
+  `'She said "Nushell is the future".'`
+- Dynamic string interpolation
+  - `$"6 x 7 = (6 * 7)"`
+  - `ls | each { |it| $"($it.name) is ($it.size)" }`
+- Bare strings
+  - `echo hello`
```

Comment:
Did I miss the backtick quotes?

Reply:
> is it said somewhere that double quotes process escapes?

Yes, that's in working_with_strings.md

Reply:
> Did I miss the backtick quotes?

Ah, yes and I didn't notice that because I didn't know they existed! What do they do?

Reply:
If you're completing paths with spaces, they'll surround those paths.  we wanted it to mean string literal but I'm not sure that ever got implemented.